### ISSUE TITLE
Pylon2mqtt on esp?

### ISSUE THREAD

**Issue by mihaigsm2003** [tomascrespo/pylon2mqtt#1]
hi, can i do script for esp32 or esp8266?

**Comment by tomascrespo** [comment]
Of course, you can. CLASSICDIY has already something done. Take a look [here](https://github.com/ClassicDIY/PylonToMQTT/wiki/5.-Flashing-the-ESP32)
Irek also has an interesting project for ESP [here](https://github.com/irekzielinski/Pylontech-Battery-Monitoring)
